src/components/Description/components/Care.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+
+export interface CareProps {
+  title: string;
+  items: Array<string>;
+}
+
+const Care = (props: CareProps) => {
+  const {title, items} = props;
+  
+  return (
+    <div className={'care'}>
+      <h3>{title}</h3>
+      <div className={'hCenterBlock'}>
+        {items && items.map( (item, i) => (
+          <p key={i} dangerouslySetInnerHTML={{ __html: item }} /> ))}
```

Comment:
Please dont use dangerouslySetInnerHTML, we are using markdown in this project for this type of this, ill will explain you how to use it, for now just use static mockup text